### PR TITLE
Provider docs update

### PR DESCRIPTION
**Pull request by beevital**

### CODE REVIEW

**Comment by lamalex** [comment]
THIS WAS SO HELPFUL THANK YOU

File: docs/index.md [hunk 1]
```diff
@@ -43,19 +41,10 @@ provider "fivetran" {
...
-
+<!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Optional
 
-- `api_key` - can be set by the environment variable `FIVETRAN_APIKEY`
-- `api_secret` - can be set by the environment variable `FIVETRAN_APISECRET`
+- `api_key` (String)
```
Comment (akankaluga) [inline docs/index.md:49]: Do we have a description of this variables somewhere?

File: fivetran/provider.go [hunk 2]
```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-var limit = 1000        // REST API response objects limit per HTTP request
-const version = "0.6.5" // Current provider version
+var limit = 1000         // REST API response objects limit per HTTP request
+const version = "0.6.12" // Current provider version
```
Comment (akankaluga) [inline fivetran/provider.go:12]: Should it be 0.6.13 already?

**Review by akankaluga — APPROVED** [review]
👍

**Pull request merged**